fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -215,6 +215,8 @@ class AltoolTransporterExecutor < TransporterExecutor
 
     private_constant :ERROR_REGEX
 
+    attr_reader :errors
```

Comment:
This _seems_ safe, it makes testing the specific error line we're adding for the `-1` return more straightforward